src/style.css

```
@@ -1,3 +1,73 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+
 body {
   margin: 0;
+  padding: 0;
+  border: 0;
+  background-color: #fff;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  width: auto;
+  background-color: #fff;
+  align-items: center;
+  margin: 0;
+  padding-right: 50px;
```

```suggestion
  padding: 0 50px;
```

and remove margin-left from logo